backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -390,8 +394,10 @@ public ResponseEntity<ResponseMessage> updateProcessDataById(@RequestBody @Valid
 
 	@PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
 	@GetMapping("/getformfields/header")
```

Comment:
Let's use this opportunity to correct the REST URL naming conventions, especially since Prabh will be integrating with the new code. If my understanding is correct, these are the column headings we see at the top. We can also change the URL accordingly which follows Rest naming conventions. 

/display-columns-names/

Reply:
I made the change as per your suggestion.